modules/local/ideel.nf

```diff
@@ -0,0 +1,24 @@
+process IDEEL {
+    label 'process_single'
+
+    container "quay.io/microbiome-informatics/ideel:1.0.0"
+
+    input:
+    tuple val(meta), val(stats)
+    val(before_after)
+    
+    output:
+    tuple val(meta), env(platform),     emit: reads
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def out = "${prefix}_${before_after}.png"
+    """
+    ideel.py ${stats} ${out}
+
```

Comment:
It's missing the "versions"

Reply:
True, thanks. Should it be the python version?

Reply:
I guess we need to rename that tool/script. It looks like proper tool, but that is just a python script that uses matplotlib 🤦‍♀️

Reply:
I think just having the script in the bin and a container with matplotlib is enough. The name overall it's fine with me.